app/presenters/concerns/add_course_to_application.rb

```diff
@@ -0,0 +1,15 @@
+module AddCourseToApplication
```

Comment:
Could we use module namespaces to imply what this presentation concern provides?
eg.
```ruby
module Presenters
  module Application
    module Course
    ...
    end
  end
end
```